Add article migrating between RaaSes (#2584)
OffchainLabs/arbitrum-docs: docs/partials/glossary/_raas.mdx

---
title: raas
key: raas
titleforSort: RaaS
---

> RaaS (Rollup as a Service) is a platform that provides the infrastructure and tools needed to deploy and operate blockchain Rollups without requiring teams to build the underlying technical infrastructure themselves. It typically includes pre-built Rollup software, node hosting, data availability solutions, and monitoring tools, allowing developers to focus on their application logic rather than the complex technical implementation of Rollup technology.

Reviewer: ```suggestion

RaaS (Rollup as a Service) is a platform that provides the necessary infrastructure and tools to deploy and operate blockchain Rollups, eliminating the need for teams to build the underlying technical infrastructure themselves. It typically includes pre-built Rollup software, node hosting, data availability solutions, and monitoring tools, allowing developers to focus on their application logic rather than the complex technical implementation of Rollup technology.
```
